### PR TITLE
Fixed image disclaimer text negatively effects SEO.

### PR DESCRIPTION
**Pull request by colourgarden**
### Ticket
- [Labeling product images"Bild zeigt evtl. Zubehör. Lieferumfang beachten!"](https://app.asana.com/0/30156186242982/1150180586264148/f)

### Description
- Google takes text nearest image as description. By moving disclaimer text to CSS pseudo element, we avoid any negative SEO effect from having false info about the image as nearest text

### CODE REVIEW

File: assets/scripts/script.js [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
   $(document).ready(function () {
-
-    $('.disclaimer-overlay').on('click', function () {
-      $tab = $('.woocommerce-tabs ul.tabs li.common_tab_tab a');
-      $tab.trigger('click');
-      $('html, body').animate({
-        scrollTop: $tab.offset().top - 230
-      }, 600);
-    });
-
```
Comment (colourgarden) [inline assets/scripts/script.js:11]: @LucaPipolo Do you remember what this code is for? Can't really make sense of it and it doesn't really seem required?

Comment (LucaPipolo) [reply assets/scripts/script.js:11]: @colourgarden, no sorry. I'm quite sure this was coming from the old itstrategen plugin but not sure which was the purpose here. 😖

File: src/Plugin.php [hunk 2]
```diff
@@ -50,11 +50,15 @@ public static function wp_enqueue_scripts() {
...
+   * Text displayed in front-end via CSS pseudo element.
+   *
    * @implements woocommerce_single_product_image_thumbnail_html
    */
   public static function woocommerce_single_product_image_thumbnail_html($content = NULL) {
-    $disclaimer = '<span class="disclaimer-overlay">' . __('Photo may show optional equipment not included in delivery.', Plugin::L10N) . '</span>';
-    return preg_replace('/(<img[^>]+>(?:<\/img>)?)/i', '$1' . $disclaimer, $content);
+    $match = 'class="woocommerce-product-gallery__image"';
+    $disclaimer = ' title="' . __('Photo may show optional equipment not included in delivery.', Plugin::L10N) . '"';
+    return str_replace($match, $match . $disclaimer, $content);
```
Comment (LucaPipolo) [inline src/Plugin.php:61]: What happens if the image thumbnail already have a title? 🤔

File: assets/scripts/script.js (hunk 1 above)
Comment (colourgarden) [reply assets/scripts/script.js:11]: Well as there won't be a `.disclaimer-overlay` in the DOM any longer, it won't have any effect anyway. It looks like it scrolls the page when the lightbox but I don't really understand the purpose of that anyway.

File: src/Plugin.php (hunk 2 above)
Comment (colourgarden) [reply src/Plugin.php:61]: `.woocommerce-product-gallery__image` is not actually the `` tag but the `<div>` wrapping the image and link. So it won't ever get a `title` added to it.
Go WooCommerce 👏

**Review by LucaPipolo — APPROVED** [review]
@colourgarden — PR approved but please, remind to bump plugin version. Thanks! :)

**Pull request merged**